adafruit_bmp280.py

```diff
@@ -30,6 +34,15 @@
 
 from micropython import const
 
+# Used only for type annotations.
```

Comment:
Since these are only used for type annotations, they should go in the try/except block below after `from typing import Optional`.  That way, the microcontroller won't need to use the RAM to import them for non-functional purposes!